python/oneflow/distributed/launch.py

```diff
@@ -108,6 +108,12 @@ def main():
     current_env["MASTER_PORT"] = str(args.master_port)
     current_env["WORLD_SIZE"] = str(dist_world_size)
 
+    if args.master_port is None or args.master_port >= 2 ** 16:
+        raise ValueError(
+            f"The port number of the rendezvous endpoint '{args.master_addr}:{args.master_port}' must be an integer "
```

Comment:
看起来 rendezvous 是 pytorch 的专用术语 https://pytorch.org/docs/stable/elastic/rendezvous.html ，我们换一个别的词怎么样，比如

```suggestion
            f"The port number of the master endpoint '{args.master_addr}:{args.master_port}' must be an integer "
```